sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -3054,11 +3054,11 @@
         "properties": {
           "status": {
             "format": "int32",
-            "description": "Method invocation result status.",
+            "description": "Method invocation result status, provided by the device.",
             "type": "integer"
           },
           "payload": {
-            "description": "Method invocation result payload.",
+            "description": "The JSON-formatted direct method result payload, up to 128kb in size; provided by the device.",
```

Comment:
What do you mean by provided by the device? Why do we need to say that? Can this come from anywhere else?

Reply:
This is to maintain parity with the description for `status` in the result; to highlight that both of these values are set by the device

Reply:
Trying to understand how this helps from user perspective? Do the SDK users understand difference between device set and application set status?

Reply:
For payload in particular the statement `provided by the device` isn't of any significance; since the payload is always set by the device. The status, on the other hand, can be set by either the device or service. (device-set and application-set mean the same thing).